test/forgery_test.exs

```diff
@@ -45,4 +45,12 @@ defmodule ForgeryTest do
     assert user1.password == "123456"
     assert user2.password == "123456"
   end
+
+  test "make_many/2 with zero amount" do
+    assert [] == MyFactory.make_many(:user, 0, %{password: "123456"})
+  end
+
+  test "make_many/2 with negative amount" do
+    assert [] == MyFactory.make_many(:user, -1, %{password: "123456"})
```

Comment:
I'm not sure if it's acceptable here to return an empty list even with not so correct amount (in this case, negative) since returning an error or raising an exception would result in an API breaking change

Reply:
Maybe we can take inspiration from `String.duplicate/2`:

```elixir
iex(9)> String.duplicate("a", 0)
""

iex(10)> String.duplicate("a", -1)
** (ArgumentError) argument error
    (stdlib) :binary.copy("a", -1)
```

Reply:
I think it's fine to raise on negative amount. We need to update the typespec to `amount :: non_neg_integer()` too.

Reply:
Ok great, thanks!